doc/modules/experimental.rst

```diff
@@ -7,8 +7,12 @@
    :no-members:
    :no-inherited-members:
 
-nilearn.experimental.surface
-----------------------------
+:mod:`nilearn.experimental.surface`
+-----------------------------------
+
+.. automodule:: nilearn.experimental.surface
+   :no-members:
+   :no-inherited-members:
```

Comment:
this is actually needed to correctly reference the module and wasn't causing the :noindex: error in #4017